CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Change Log
 
+## 2.2
+
+This release is mostly a maintenance release which fixes bugs and regressions.
+It also unblocks significant future feature development.
+
+### Workaround removals
+
+- Remove old `force_bindless_texel_buffer` workaround with
+  a more correct and performant workaround.
```

Comment:
s/Remove/Replace
s/workaround/implementation since it's always on now

Reply:
Fixed.